sdk/src/test/java/io/outbound/sdk/OutboundTest.java

```diff
@@ -0,0 +1,16 @@
+package io.outbound.sdk;
+
+import static org.junit.Assert.*;
+
+public class OutboundTest {
```

Comment:
Any sense in committing this file?

Reply:
I honestly don't even remember adding this directory to the project. Possibly a remnant of trying to mock my way to unit testable victory